packages/api/src/command/medical/document/document-query.ts

```diff
@@ -41,12 +41,14 @@ export async function queryDocumentsAcrossHIEs({
   patientId,
   facilityId,
   override,
+  cxPayload,
```

Comment:
I'd call this `cxRequestMeta`  or `cxRequestMetadata` as payload doesn't indicate much what that's about. Here and everywhere else.